.eslintrc.js

```diff
@@ -137,6 +138,7 @@ module.exports = {
     'prefer-destructuring': 'off',
     'function-paren-newline': ['error', 'consistent'],
     'implicit-arrow-linebreak': 'off',
+    'no-multiple-empty-lines': 'off',
```

Comment:
There is a PR merged that fixes all of our instances of multiple empty lines, so this line can be safely removed.

Reply:
b1d10f6b3